language/sygus.tex

```diff
@@ -2310,17 +2310,17 @@ \section{Calling Oracles}%
 
 In the following, we formalize the interface between the
 synthesis solver and an external binary, which we call an \emph{oracle}.
-The synthesis solver can query an oracle using text files with the extension
-\textit{.oracle}.
+An oracle can be queried either via command line (default) 
+or using text files with the extension
+\textit{.oracle}. If the oracle should be queried via text files, it is declared with the attribute $\texttt{:file}$.
```

Comment:
Actually, revisiting this, do we really need to have any requirement on the extension here?  Or maybe we could say `.oracle` is a recommended extension but not requirement?

Reply:
ok, i've changed the references to text files to .query as extension, and removed the requirement that says they have to have that extension. i.e., it can be any text file, but we happen to have named ours .query

Reply:
Sounds good to me; thanks!